src/components/admin/activities/manage-activities.js

```diff
@@ -1,10 +1,106 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
-
-export default function ManageActivities() {
-  return (
-    <Typography paragraph>
-      The 'Manage Activities' Page comming soon...
-    </Typography>
-  );
+import { DataGrid } from '@material-ui/data-grid';
+import { dummyData, columns } from './activities-const'
+import SearchActivities  from './search-activities'
+import ActivityTypeSelect from './activity-type-select';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import AddEditActivity from './add-edit-activity';
+
+class ManageActivities extends React.Component {
+
+    constructor(props) {
+      super(props);
+
+      this.state = {
+          activityRows: [],
+          filteredActivities: [],
+          searchTerm: "",
+          typeSearchTerm: '',
+          isAddEditActivityModalOpen: false
+      };
+
+      this.filterBySearch = this.filterBySearch.bind(this);
+      this.filterByType = this.filterByType.bind(this);
+      this.filterList = this.filterList.bind(this);
+      this.handleSaveActivityToggle = this.handleSaveActivityToggle.bind(this);
+  }
+
+    componentDidMount() {
+
+      const activityRows = dummyData;
+
+      this.setState({
+        activityRows: activityRows,
+        filteredActivities: activityRows
+      });
+  }
+
+  filterBySearch(term) {
+    this.setState({
+      searchTerm: term
+    });
+    this.filterList(term, this.state.typeSearchTerm);
+  }
+
+  filterByType(term) {
+    this.setState({
+      typeSearchTerm: term
+    });
+    this.filterList(this.state.searchTerm, term);
+  }
+
+  filterList = (searchTerm, searchByTypeterm) => {

```

Comment:
@simonyangevorg jan, the same change needs to be done here. 

You've bound this function in constructor
`this.filterList = this.filterList.bind(this);` 
Ne need of arrow function